src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyAliasCmdlet.cs

```diff
@@ -116,12 +115,12 @@ private bool IsMatch(string input, string match)
         private IEnumerable<Provider> GetAllProviders()
         {
             var returnList = new List<Provider>();
-            var tempResult = this.ResourceManagerSdkClient.ResourceManagementClient.Providers.List(expand: "resourceTypes/aliases");
+            var tempResult = this.ResourceManagerSdkClient.ResourceManagementClient.Providers.ListAtTenantScope(expand: "resourceTypes/aliases");
```

Comment:
No recorded sessionrecords needed to be updated for tests after making this change?

Reply:
I made a recorded session file and added it. Need to investigate why it isn't showing here.

Reply:
I expected it to, but the tests all passed on playback.

Reply:
The tests for this cmdlet aren't live scenario tests so it doesn't need recordings.